virtool/startup.py

```diff
@@ -47,6 +49,12 @@
 logger = logging.getLogger("startup")
 
 
+@dataclass
+class B2CInfo:
```

Comment:
"Info" is not accurate. `ClientApplication` is a collection of methods and attributes abstracting access to the B2C API.

```suggestion
class B2C:
```